src/csvcubed/writers/helpers/qbwriter/dsdtordfmodelshelper.py

```diff
@@ -301,17 +302,33 @@ def _get_qb_component_specs_for_col(
                 self._get_qb_attribute_specification(column_name_uri_safe, component)
             ]
         elif isinstance(component, QbMultiUnits):
-            return [self._get_qb_units_column_specification(column_name_uri_safe)]
+            return self._get_qb_units_column_specification(column_name_uri_safe)
         elif isinstance(component, QbMultiMeasureDimension):
             return self._get_qb_measure_dimension_specifications(component)
         elif isinstance(component, QbObservationValue):
             return self._get_qb_obs_val_specifications(component)
         else:
             raise TypeError(f"Unhandled component type {type(component)}")
 
+    """
```

Comment:
This comment should be against the field we added (which you've now moved up).

Reply:
moved it